public/contribute/index_zh.ejs

```diff
@@ -1,3 +1,19 @@
+<%
```

Comment:
Is it possible to put these functions in a utils.ejs and include them instead of putting them at the top of multiple files?
<%- include utils.ejs %>